### PR TITLE
add job/periodic-promoted-image-governor

### PR DESCRIPTION
**Pull request by hongkailiu**
https://issues.redhat.com/browse/DPTP-1766

Wait for https://github.com/openshift/ci-tools/pull/2163

### CODE REVIEW

**Comment by hongkailiu** [comment]
/hold

need rehearsal to pass.

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -1976,3 +1976,30 @@ periodics:
...
+  name: periodic-promoted-image-governor
+  spec:
+    containers:
+    - command:
+      - /usr/bin/promoted-image-governor
+      - --ci-operator-config-path=ci-operator/config
+      - --release-controller-mirror-config-dir=core-services/release-controller/_releases
+      - --ignored-image-stream-tags='^ocp\S*/\S+:machine-os-content$'
+      - --ignored-image-stream-tags='^openshift/origin-v3.11:'
+      - --dry-run=true
```
Comment (hongkailiu) [inline ci-operator/jobs/infra-periodics.yaml:1998]: I will change it to `false` after verifying the ignored tags goes through.

**Review by petr-muller — APPROVED** [review]
/hold

**Comment by hongkailiu** [comment]
/hold cancel

image job passed.

**Pull request merged**